okhttp/src/main/kotlin/okhttp3/internal/cache/DiskLruCache.kt

```diff
@@ -20,13 +20,14 @@ import okhttp3.internal.cache.DiskLruCache.Editor
 import okhttp3.internal.closeQuietly
 import okhttp3.internal.concurrent.Task
 import okhttp3.internal.concurrent.TaskRunner
-import okhttp3.internal.io.FileSystem
 import okhttp3.internal.isCivilized
 import okhttp3.internal.okHttpName
 import okhttp3.internal.platform.Platform
 import okhttp3.internal.platform.Platform.Companion.WARN
 import okio.*
```

Comment:
off-topic: IntelliJ has really become bad at Kotlin imports

Reply:
I think they are converging on .editorconfig.  They allowed using it for a few simple things like import length, but I believe it's presence in newer versions is very impactful.

IMHO The solution is ditching the square styles import and diving into .editorconfig